### DISCUSSION TITLE
Version 2.2.1

### DISCUSSION THREAD

**Discussion by vkbo** [vkbo/novelWriter#1673] [General]
### Release Notes

This is a patch release that fixes an issue where the Project View would sometimes switch to the Novel View when a new item was created. This patch also includes updated translations for German and Chinese.

### Detailed Changelog

**Bugfixes**

* Fix a bug in the toggle for the tree view on the left would trigger each time a new project item was created. Issue #1649. PR #1648.
* Fix an issue where multiple Manuscript and Writing Stats dialogs could be opened. PR #1671.

**Internationalisation**

* Updated German translation, by @HeyMyian. PR #1666.
* Updated Chinese translation, by @ruixuan658 and @longqzh. PR #1666.

<hr /><em>This discussion was created from the release <a href='https://github.com/vkbo/novelWriter/releases/tag/v2.2.1'>Version 2.2.1</a>.</em>

**Comment by Koldunya81** [comment]
lol thank you so much. I'm new to novelWriter and wasn't sure if that 1st thing was a bug or a feature, and had been meaning to look around to see. I'd become very quick at ctrl-t every time I added a new scene XD

**Reply by vkbo** [reply]
Yeah, the add new document/scene/whatever was followed by a focus switch to the project tree, but when I changed the focus switch to a toggle feature in 2.2, it messed it all up. Sorry it took so long to fix it. I didn't even notice it!
